servo-core/src/main/java/com/netflix/servo/DefaultMonitorRegistry.java

```diff
@@ -118,6 +118,28 @@ private static ObjectNameMapper getObjectNameMapper(Properties props) {
 
     return mapper;
   }
+  
+  private static Properties loadProps() {
+    String registryClassProp = System.getProperty(REGISTRY_CLASS_PROP);
+    String registryNameProp = System.getProperty(REGISTRY_NAME_PROP);
+    String defaultRegistryName = System.getProperty(DEFAULT_REGISTRY_NAME);
```

Comment:
Sorry missed this on the previous pass.

`DEFAULT_REGISTRY_NAME` is the default value for `REGISTRY_NAME_PROP` so this line should be removed and line 124 should be
```
  String registryNameProp = System.getProperty(REGISTRY_NAME_PROP, DEFAULT_REGISTRY_NAME);
```

Reply:
Line 73 is still checking for props.getProperty(REGISTRY_NAME_PROP, DEFAULT_REGISTRY_NAME). I didn't want to change the implementation of the method itself keeping it to minimal and straight. If i remove line 125, there will be a property missing DEFAULT_REGISTRY_NAME when it checks in Line 73

I can do this instead

```
 private static Properties loadProps() {
    String registryClassProp = System.getProperty(REGISTRY_CLASS_PROP);
    String registryNameProp = System.getProperty(REGISTRY_NAME_PROP, DEFAULT_REGISTRY_NAME);
    String registryJmxNameProp = System.getProperty(REGISTRY_JMX_NAME_PROP);
   
    Properties props = new Properties();
    if (registryClassProp != null) {
      props.setProperty(REGISTRY_CLASS_PROP, registryClassProp);
    }
    if (registryNameProp != null) {
      props.setProperty(REGISTRY_NAME_PROP, registryNameProp);
      props.setProperty(DEFAULT_REGISTRY_NAME, registryNameProp);
    }
    if (registryJmxNameProp != null) {
      props.setProperty(REGISTRY_JMX_NAME_PROP, registryJmxNameProp);
    }
    return props;
  }
```

Reply:
> If i remove line 125, there will be a property missing DEFAULT_REGISTRY_NAME when it checks in Line 73

In line 73 it is checking for `REGISTRY_NAME_PROP` and returns `DEFAULT_REGISTRY_NAME` if the property is not set. 

In terms of keeping the previous behavior, it should be:

```java
  private static Properties loadProps() {
    String registryClassProp = System.getProperty(REGISTRY_CLASS_PROP);
    String registryNameProp = System.getProperty(REGISTRY_NAME_PROP);
    String registryJmxNameProp = System.getProperty(REGISTRY_JMX_NAME_PROP);
   
    Properties props = new Properties();
    if (registryClassProp != null) {
      props.setProperty(REGISTRY_CLASS_PROP, registryClassProp);
    }
    if (registryNameProp != null) {
      props.setProperty(REGISTRY_NAME_PROP, registryNameProp);
    }
    if (registryJmxNameProp != null) {
      props.setProperty(REGISTRY_JMX_NAME_PROP, registryJmxNameProp);
    }
    return props;
  }
```